jenkins/bootstrap.py

```diff
@@ -210,7 +210,7 @@ def random_sleep(attempt):
     time.sleep(random.random() + attempt ** 2)
 
 
-def checkout(call, repo, branch, pull, ssh='', git_cache='', clean=False):
+def checkout(call, repo, repo_path, branch, pull, ssh='', git_cache='', clean=False):
```

Comment:
consider update the doc string, i.e. what's `repo`, what's `repo_path`, in case we need to bikeshed this again
/shrug

Reply:
the doc string doesn't cover any of this :|
updating.